distribution.pm

```diff
@@ -116,10 +116,11 @@ Deprecated mode
 Run I<$cmd> (by assuming the console prompt and typing the command). After that, echo
 hashed command to serial line and wait for it in order to detect execution is finished.
 To avoid waiting, use I<$timeout> 0.
+In case you have mistyping problems lower I<$typing_speed> which sets I<max_interval> in C<type_string>
 
 Use C<output> to add a description or a comment of the $cmd.
 
-Use C<quiet> to avoid recording serial_results.
+    Use C<quiet> to avoid recording serial_results.
```

Comment:
unrelated change

Reply:
As mentioned by Oli, this should be split in a separate commit.

Reply:
please attend this comment in your changes as well